converter/bodies.py

```diff
@@ -0,0 +1,91 @@
+from dataclasses import dataclass
+from abc import ABC
+
+
+@dataclass(frozen=True)
+class Body(ABC):
+    """Abstract body, bodies are used to define geometries."""
+
+    uuid: str
+    x_offset: float
+    y_offset: float
+    z_offset: float
+    x_rotation: float
+    y_rotation: float
+    z_rotation: float
+
+
+@dataclass(frozen=True)
+class SphereBody(Body):
```

Comment:
I'd use simply `Sphere` here.
Keep in mind that in TOPAS we will have sphere component, to which we will assign material.
So we will need to deal with two types of Spheres: SHIELD-HIT12A like bodies (without material) and TOPAS like geometry components with materials assigned.

Reply:
I'm not sure what you mean by "using `Sphere`", but when it comes to the two types of spheres, SHIELD-HIT12A will have a zone class containing a list of figures that define it. TOPAS figures will be composed with a material.